dataactbroker/handlers/fileHandler.py

```diff
@@ -200,9 +200,11 @@ def finalize(self):
             # Compare user ID with user who submitted job, if no match return 400
             job = self.jobManager.getJobById(jobId)
             submission = self.jobManager.getSubmissionForJob(job)
-            if(submission.user_id != LoginSession.getName(session)):
+            # Check that user's agency matches submission cgac_code or user id of submission
+            userId = LoginSession.getName(session)
+            if(submission.user_id != userId and submission.cgac_code != self.interfaces.userDb.getUserByUID(userId).cgac_code):
```

Comment:
Good point, I modified it to allow "SYS" users to finalize any job.  I believe we've been using website admin and agency admin for permissions on the route decorator, which for this route any user should be able to reach the route.